src/iap.ts

```diff
@@ -576,7 +576,7 @@ const App = () => {
 
 export const requestPurchase = (
   request: RequestPurchase,
-): Promise<ProductPurchase | ProductPurchase[] | void> =>
+): Promise<ProductPurchase | void> =>
```

Comment:
@ddowoo The return value could be an array so I think you should revert this changes.

Reference https://developer.apple.com/forums/thread/6431#14831